build.gradle

```diff
@@ -23,7 +23,7 @@ buildscript {
         binaryCompatibilityValidator = '0.2.4'
         detektVersion = '1.14.0'
         dokkaVersion = '1.4.20'
-        ktLintVersion = '0.39.0'
+        ktLintVersion = '0.40.0'
         ktLintGradleVersion = '9.4.0'
```

Comment:
Should we also bump this to `9.4.2`?

Reply:
Right, forgot about plugin. However, I see 9.4.1 as the latest one. 
https://www.github.com/JLLeitschuh/ktlint-gradle/tree/master/CHANGELOG.md

Was 9.4.2 a typo?

Reply:
Yup my bad 👍